File: php/examples/delete_payment_methods.php
<?php
require_once 'base_request.php';

// Example of DELETE /payment-methods/:payment_method_id

$debug = true;
$payment_method_id = $_ENV['PAYMENT_METHOD_ID'];
$path = '/payment-methods/' . $payment_method_id;

send_request('DELETE', $path, '', '');

File: csharp/examples/DeletePaymentMethods.cs
using System;
namespace Examples.examples
{
    // Example of DELETE /payment-methods/:payment_method_id

    public class DeletePaymentMethods : BaseRequest
    {
        private string _paymentMethodId;

        public DeletePaymentMethods()
		{
            dotenv.net.DotEnv.Load();
            _paymentMethodId = Environment.GetEnvironmentVariable("PAYMENT_METHOD_ID");
            Console.WriteLine($"DeletePaymentMethods: _paymentMethodId={_paymentMethodId}");
        }

        public override void SendRequest()
        {
            DeleteRequest($"payment-methods/{_paymentMethodId}");
        }
    }
}

